src/api/appUsers.js

```diff
@@ -1,23 +1,69 @@
-import BaseAPI from './base';
-import http from '../utils/http';
+import { BaseAPI } from './base';
+import { http } from '../utils/http';
 
-export default class AppUsersAPI extends BaseAPI {
-  retrieve(userId, auth) {
-    const url = this.getFullURL('appUsers', userId);
+
+/**
+ * Init API properties
+ * @typedef InitProps
+ */
+
+/**
+ * @class AppUsersAPI
+ * @extends BaseAPI
+ */
+export class AppUsersAPI extends BaseAPI {
+
+  /**
+   * Initializes the conversation for a user
+   * @param  {InitProps} props
+   * @param  {AuthCredentials} auth
+   * @return {APIResponse}
+   */
+  init(props, auth) {
+    const url = this.getFullURL('init');
+    return this.getAuthenticationHeaders(auth).then((authHeaders) => {
```

Comment:
@alavers  Hmm I've been playing with this a bit and I think it defeats the whole discussion we had about hiding some parts of the API.. having something on `this` would mean you could do `smoochInstance.appUsers.post()` and do anything..